doc/data/messages/t/too-many-lines/bad.py

```diff
@@ -0,0 +1,18 @@
+def leap_year(year): # [too-many-lines]
+    """ Function used to determine whether a given year is a leap year """
+    if year % 400 == 0:
+        return True
+    if year % 100 == 0:
+        return False
+    if year % 4 == 0:
+        return True
+    else:
+        return False
+
+for year in range(1990, 2000):
```

Comment:
We should not use the loop in bad.py or we can't have the same code with less Line in good.py easily

Reply:
Ok, I removed the for loop.